### PR TITLE
Add analysis

### PR DESCRIPTION
**Pull request by dtingey**
This is to add a generative analysis and response recommender model.

### CODE REVIEW

**Comment by dtingey** [comment]
@Isaacwilliam4 @brightlightkim @JollyJiant Please review/approve 😄

File: insyt/inference.py [hunk 1]
```diff
@@ -2,5 +2,5 @@
 
 # TODO: replace with actual model (Isaac/Bronze) Maybe move to a different directory if necessary
```
Comment (brightlightkim) [inline insyt/inference.py:3]: Thank you for putting down this TODO list as a comment that we can work on it.

File: insyt/ollama.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+def run_model(file_line, previous_lines, classification):
+    """Run the analysis model given the line and context"""
+    log.debug(f"Running analysis model on line: {file_line}")
+    log.debug(f"Previous lines: {previous_lines}")
+    log.debug(f"Classification: {classification}")
+
+    prompt = f"The following line has been classified as '{classification}':\n\n```{file_line}```\n\n" + \
+        "Explain why it was classified as such and how I should respond."
+    
+    response = ollama.chat(model='insyt', messages=[{'role': 'user', 'content': prompt}])
```
Comment (brightlightkim) [inline insyt/ollama.py:15]: Interesting that we are using the Generative model with LLaMA model. Good job.

**Review by brightlightkim — APPROVED** [review]
Great job on organizing the code and the comment. :)

**Pull request merged**